CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by a [Code of Conduct](code_of_conduct.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [quack@duckdb.org](mailto:quack@duckdb.org).
+
+## Eligibility
+
+Before submitting a contribution, please check whether your contribution is eligible.
+
+1. Before creating a new page, please [search the existing documentation](https://duckdb.org/docs/search) for similar pages.
+2. In general, guides for third-party tools using DuckDB should not be included in the DuckDB documentation. Rather, these tools and their documentation should be collected in the [`awesome-duckdb` community repository](https://github.com/davidgasquez/awesome-duckdb).
+
+## Adding a new page
+
+Thank you for contributing to the DuckDB documentation!
+
+Each new page requires at least 2 edits:
+* The creation of a new Markdown page with the documentation. Please follow the format of another `.md` file in the `docs` folder.
+* Add a link to the new page within `_data/menu_docs_dev.json`. This populates the dropdown menus.
+
+The addition of a new guide requires one additional edit:
+* Add a link to the new page within the Guides landing page: `docs/guides/index.md`
+
+**Please test your changes using the steps listed in the [Building](building.md) guide.**
+
+**When creating a PR, please check the box to "Allow edits from maintainers".**
+
+## Formatting
+
+### Syntax
```

Comment:
Optional, but would be worth mentioning that quote blocks are styled specially to appear as info boxes